changelog/json_query.dd

```diff
@@ -0,0 +1,23 @@
+Added the `-Xq=...` option to support limiting JSON output to a set of fields.
+
+Using the `-Xq=...` option limits the JSON output to a set of fields, i.e.
+---
```

Comment:
The `CONSOLE` macro is specifically made for this:

```
$(CONSOLE dmd -Xq=compilerInfo,semantics)
```

see e.g. https://dlang.org/changelog/pending.html#deprecation-dylib